platform-tooling-support-tests/projects/graalvm-starter/build.gradle.kts

```diff
@@ -31,8 +31,6 @@ tasks.test {
 graalvmNative {
 	binaries {
 		named("test") {
-			buildArgs.add("--initialize-at-build-time=org.junit.platform.launcher.core.LauncherConfig")
-			buildArgs.add("--initialize-at-build-time=org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter")
```

Comment:
Even though recent versions of Native Build Tools register those as well, I don't think we should remove this configuration.

- see https://github.com/junit-team/junit5/issues/3040

Reply:
Why should we keep it? The reason for adding this here was because Native Build Tools didn't already configure those. I think adding a [`native-image.properties`](https://www.graalvm.org/22.1/reference-manual/native-image/BuildConfiguration/#embedding-a-configuration-file) file is a separate issue.

Reply:
Ahhh... yes, you're right.

This is just our test build script. I apologize for not thinking that through.

Please just disregard that comment, and I'll _hide_ it. 😉